tensorflow/python/training/moving_averages_test.py

```diff
@@ -271,7 +271,8 @@ def testAverageVariablesDeviceAssignment(self):
       v0 = tf.Variable(10.0, name="v0")
     with tf.device("/job:dev_v1"):
       v1 = gen_state_ops._variable(shape=[1], dtype=tf.float32, 
-          name="v1", set_shape=True, container="", shared_name="")
+          name="v1", container="", shared_name="")
+      v1.set_shape([1])
```

Comment:
Only this test need to set shape, so I don't `set_shape` for other tests. 
I have run all test scripts at local, and I think they would pass the test this time.